docker/dockerfile

```
@@ -79,9 +83,17 @@ WORKDIR /
 # install doxygen
 RUN dpkg -i ${INSTALLER_PATH}/${DOXYGEN}
 
-# install PackChk
-RUN cp ${INSTALLER_PATH}/PackChk /usr/local/bin/PackChk
-RUN chmod +x /usr/local/bin/PackChk
+# install CMSIS-Toolbox
+ENV CMSIS_PACK_ROOT=${HOME}/.packs
+RUN wget https://github.com/Open-CMSIS-Pack/cmsis-toolbox/releases/download/1.2.0/cmsis-toolbox-linux64.tar.gz && \
+    tar -xvf cmsis-toolbox-linux64.tar.gz && \
+    sed -i -e 's/set(TOOLCHAIN_ROOT "[^"]*")/set(TOOLCHAIN_ROOT "$ENV{CI_ARMCC5_TOOLCHAIN_ROOT}")/' cmsis-toolbox-linux64/etc/AC5.5.6.7.cmake && \
+    sed -i -e 's/set(TOOLCHAIN_ROOT "[^"]*")/set(TOOLCHAIN_ROOT "$ENV{CI_ARMCC6_TOOLCHAIN_ROOT}")/' cmsis-toolbox-linux64/etc/AC6.6.18.0.cmake && \
```

Talking to @JonatanAntoni it seems his expectation was that the shell would replace the environment variable `CI_<compiler>_TOOLCHAIN_ROOT` before writing to the `.cmake` file. To achieve this goal instead of using the cmake `ENV` operator the `sed` command could be rewritten for example in this way:
```suggestion
    sed -i -e 's|set(TOOLCHAIN_ROOT "[^"]*")|set(TOOLCHAIN_ROOT "'${CI_ARMCC5_TOOLCHAIN_ROOT}'")|' cmsis-toolbox-linux64/etc/AC6.6.18.0.cmake && \
```